README.md

```diff
@@ -69,6 +69,10 @@ Options:
 - `maxHeaderSize`, the maximum length of request headers in bytes. 
   Default: `16384` (16KiB)
 
+- `headersTimeout`, the amount of time the parser will wait to receive the complete
+  HTTP headers (Node 12 and above only).
+  Default: `60e3` milliseconds (30s).
```

Comment:
Maybe change to 30s to be consistent with our other timeouts?

Reply:
I agree